### ISSUE TITLE
Doesn't work with GitHub Desktop 

### ISSUE THREAD

**Issue by SaadBazaz** [Escape-Technologies/mookme#126]
Works fine on macOS when using Git CLI, but fails on Windows when using GitHub Desktop.

**Comment by jahansoft** [comment]
Exactly, I Agree, I am using GitHub Desktop, I have download it from [Here](https://desktop.github.com/).
